### PR TITLE
text color changed

### LINKED ISSUE #4159: Change the text color
### Description

on https://layer5.io/company/about the color of text in a section is black. It should be changed to white
### Screenshots

image

### Environment:
- Host OS: 
- Browser: 

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Mohith234**
**Description**

This PR fixes #4159 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit f6c7c98bda0006bc8dcd55ef6acd119c42137633 at: https://644cc72163f07565be818ea6--layer5.netlify.app

**Comment by Mohith234** [comment]
Why I am getting this error ?

**Review by hirentimbadiya — APPROVED** [review]
All good @Mohith234 , Thanks for your contribution 🎉

File: src/sections/Company/Layer5-statement/statement.style.js [hunk 1]
```diff
@@ -37,6 +37,9 @@ const BannerSectionWrapper = styled.section`
         color: ${(props) => props.theme.white};
         margin-bottom: 1rem;
     }
+    span.span-text{
+        color: white;
```
Comment (thoughtlessmind) [inline src/sections/Company/Layer5-statement/statement.style.js:41]:
```suggestion
        color: ${(props) => props.theme.white};
```
what do you think about this @Mohith234 ?

Comment (Mohith234) [reply src/sections/Company/Layer5-statement/statement.style.js:41]: @thoughtlessmind The background image color doesn't change when the theme changes. I think the text color is also independent of the theme.

Do I still need to change?

**Comment by Mohith234** [comment]
@thoughtlessmind @hirentimbadiya Please do check if I need to make any changes

File: src/sections/Company/Layer5-statement/statement.style.js (hunk 1 above)
Comment (thoughtlessmind) [reply src/sections/Company/Layer5-statement/statement.style.js:41]: Hey @Mohith234
sorry for the delay. I've less context on that, maybe someone with more context on that can clarify that
thanks

Comment (Shivam-AfA) [reply src/sections/Company/Layer5-statement/statement.style.js:41]: @Mohith234 using `color: ${(props) => props.theme.white};` would be fine.

File: src/sections/Company/Layer5-statement/statement.style.js [hunk 2]
```diff
@@ -37,6 +37,9 @@ const BannerSectionWrapper = styled.section`
         color: ${(props) => props.theme.white};
         margin-bottom: 1rem;
```
Comment (Shivam-AfA) [inline src/sections/Company/Layer5-statement/statement.style.js:38]: please change this to `margin-bottom: 2rem;`

**Comment by l5io** [comment]
🚀 Preview for commit 39f38838876e77a281f7a99ab9a0956cf51637d6 at: https://645209d5ab7ea448b6804ba1--layer5.netlify.app

**Comment by Shivam-AfA** [comment]
Thanks for this @Mohith234 .

**Pull request merged**